index.js

```diff
@@ -826,6 +888,18 @@
     return function(x) { return f(chain(x))(x); };
   }
 
+  //  Function$prototype$alt :: (x -> a) ~> (x -> a) -> x -> a
+  function Function$prototype$alt(f) {
+    var alt_ = this;
+    return function(x) { return alt(alt_(x), f(x)); };
+  }
+
+  //  Function$prototype$concat :: (x -> a) ~> (x -> a) -> x -> a
+  function Function$prototype$concat(f) {
+    var semigroup = this;
+    return function(x) { return concat(semigroup(x), f(x)); };
+  }
+
```

Comment:
@i-am-tom, would you mind opening two more pull requests? One for `Function#fantasy-land/alt` and one for `Function#fantasy-land/concat`. It would be nice to review and merge these changes separately as they're unrelated to #43.